lite/api/paddle_api.h

```diff
@@ -295,6 +295,9 @@ class LITE_API CxxConfig : public ConfigBase {
   // XPU only, set the size of the workspace memory from L3 cache for the
   // current thread.
   void set_xpu_workspace_l3_size_per_thread(int l3_size = 0xfffc00);
+
+  void set_xpu_conv_autotune(bool autotune = true);
```

Comment:
```suggestion
  void set_xpu_conv_autotune(bool autotune = true, std::string autotune_file = "");
```
既然可以不从环境变量获得autotune的信息，那么autotune_file也一起设置在接口里面比较合适一些

Reply:
Done, 这两个参数可以分开设置